packages/cfd/src/Stores/connect.js

```diff
@@ -17,7 +17,8 @@ function injectStorePropsToComponent(propsToSelectFn, BaseComponent) {
             ObservedComponent = FunctionalWrapperComponent;
         }
 
-        return useObserver(() => ObservedComponent({ ...own_props, ...propsToSelectFn(store, own_props) }));
+        const context = own_props.context || store;
```

Comment:
Please add a comment that this is a temporary fix. We need to find out a better way to fix it.